roles/taca/tasks/main.yml

```diff
@@ -0,0 +1,27 @@
+- name: Fetch TACA-ngi-pipeline from GitHub
+  git: repo={{ taca_ngi_repo }}
+       dest={{ tca_ngi_dest }}
+       version={{ taca_ngi_version }}
+       force=yes
+  tags: taca
+
+- name: Possibly setup virtual environment and install dependencies 
+  shell: conda create -n NGI pip sqlalchemy 
+  when: not ngi_venv_exists.stdout 
```

Comment:
If you're going down this route then this will require that the ngi_pipeline role has been run before, otherwise this task will fail to find `ngi_venv_exists`.